addons/html_builder/static/src/plugins/vertical_alignment_option_plugin.js

```diff
@@ -24,6 +24,33 @@ export class VerticalAlignmentOptionPlugin extends Plugin {
             SetVerticalAlignmentAction,
         },
     };
+
+    setup() {
+        this.upgradeContainers();
+    }
+
+    // TODO: Remove once data-vxml="number" snippet comparison is restored.
+    upgradeContainers() {
+        // --- 1) Handle .s_cards_soft ---
+        for (const el of this.document.querySelectorAll(".s_cards_soft .s_card")) {
+            el.classList.add("h-100");
+        }
+
+        // --- 2) Handle .s_cards_grid ---
+        for (const cardEl of this.document.querySelectorAll(".s_cards_grid .s_card")) {
+            cardEl.classList.add("h-100");
```

Comment:
This turns the existing blocks that were "Align Top" into "Stretch to equal height".
The align top option-related classes also need to be added in order for the existing layout to not change.